api/cpp/CMakeLists.txt

```diff
@@ -431,8 +431,12 @@ if(SLINT_BUILD_RUNTIME)
         endforeach()
 
         get_property(_SLINT_STYLE GLOBAL PROPERTY SLINT_STYLE)
-        get_target_property(_slint_compiler_location slint-compiler IMPORTED_LOCATION)
-        cmake_path(GET _slint_compiler_location FILENAME SLINT_COMPILER_FILE_NAME)
+        if (TARGET slint-compiler)
+            get_target_property(_slint_compiler_location slint-compiler IMPORTED_LOCATION)
+            cmake_path(GET _slint_compiler_location FILENAME SLINT_COMPILER_FILE_NAME)
+        endif()
+
+        set(SLINT_VERSION 1.8.0)
```

Comment:
can we derive this value from `CPACK_PACKAGE_VERSION_MAJOR` and so on?

And use this variable a few line down as well so we don't repeat the version too much.

Reply:
I thought of using the project version, but that one is wrong and harder to change from the version script.

I agree though that the CPack version should be reused. I can spend a bit more time on unifying the three then.

Reply:
why is the project version wrong?

Reply:
It says 1.0.0 and that's the bit i thought of changing in the context of the "spending more time" statement.